src/mode.js

```diff
@@ -173,5 +173,10 @@ export function getFullVersion_(win, isLocalDev) {
     return win.AMP_CONFIG.v;
   }
 
-  return version;
+  // Currently `$internalRuntimeVersion$` and thus `mode.version` contain only
+  // major version. The full version however must also carry the minor version.
+  // We will default to production default `01` minor version for now.
+  // TODO(erwinmombay): decide whether $internalRuntimeVersion$ should contain
+  // minor version.
+  return `01${version}`;
```

Comment:
I took to using two types, `AmpVersion` and `RtvVersion` in the SW to help catch issues where I used a non-prefixed value in a place that requires a prefix. Maybe we should do the same here to catch these kinds of issues?